### PR TITLE
fix golint install message

### LINKED ISSUE #135: verify-golint recommends incorrect go get
/kind bug

**What steps did you take and what happened:**
```
$ make                                                                   
go generate ./pkg/... ./cmd/...                        
go fmt ./pkg/... ./cmd/...                                                             
go vet ./pkg/... ./cmd/...                                        
hack/verify-golint.sh                                  
Can not find golint, install with:       
go get -u github.com/golang/lint/golint                
Makefile:52: recipe for target 'lint' failed
make: *** [lint] Error 1
```

**What did you expect to happen:**
go get golang.org/x/lint/golint
per https://github.com/golang/lint/issues/415

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Istio Version:
- Knative Version:
- KFServing Version:
- Kubeflow version:
- Minikube version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

### PR DESCRIPTION
**Pull request by rakelkar**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #135 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/136)

### CODE REVIEW

**Comment by rakelkar** [comment]
@gyliu513 FYI

File: hack/verify-golint.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 if ! which golint > /dev/null; then
   echo 'Can not find golint, install with:'
-  echo 'go get -u github.com/golang/lint/golint'
+  echo 'go get -u go get golang.org/x/lint/golint'
```
Comment (gyliu513) [inline hack/verify-golint.sh]: I think here should be 

```
go get -u golang.org/x/lint/golint
```

**Pull request merged**